torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4196,6 +4212,12 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            supports_inplace_autograd=True,
            safe_casts_outputs=True,
            sample_inputs_func=sample_inputs_xlogy),
+    OpInfo('logsumexp',
+           dtypes=floating_types_and(torch.bfloat16),
+           dtypesIfCUDA=all_types_and(torch.bfloat16, torch.half, torch.int64),
```

Comment:
Shouldn't the integer dtypes be removed for CUDA?

Would you file an issue for the odd error when passing integer tensors to logsumexp along with a snippet to reproduce it, too?